### PR TITLE
 feature/MAC-7

### PR DESCRIPTION
**Pull request by adiputrachristian**
#Rising Cases Table
- rising table view
- rising row table
- rising header table
- extension

### CODE REVIEW

File: Natics/Extension/Extension+NSTableView.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//  Natics
+//
+//  Created by Christian Adiputra on 03/11/21.
+//
+
+import Foundation
+import SwiftUI
+
+extension NSTableView {
+    open override func viewDidMoveToWindow() {
```
Comment (handharr) [inline Natics/Extension/Extension+NSTableView.swift:12]: Ini override functionnya buat apa ya?

Comment (adiputrachristian) [reply Natics/Extension/Extension+NSTableView.swift:12]: buat remove warna background list

Comment (adiputrachristian) [reply Natics/Extension/Extension+NSTableView.swift:12]: Screen Shot 2021-11-06 at 10 03 10

Comment (adiputrachristian) [reply Natics/Extension/Extension+NSTableView.swift:12]: Kalo itu diremove, warnanya backgroundnya gamau berubah. udah diclear background tetep ga bs

**Pull request closed without merging**